mycroft/skills/mycroft_skill/mycroft_skill.py

```diff
@@ -152,7 +158,8 @@ def __init__(self, name=None, bus=None, use_settings=True):
 
         #: Filesystem access to skill specific folder.
         #: See mycroft.filesystem for details.
-        self.file_system = FileSystemAccess(join('skills', self.name))
+        # fully initialized when self.skill_id is set
+        self.file_system = FileSystemAccess(get_temp_path(self.name))
```

Comment:
right, this is confusing because it was split into #28 

i made filesystem use skill_id instead of name, but skill_id is not set between `__init__` and ìnitialize`

This is a precaution more than anything else, the homescreen skill was using self.filesystem in `__init__` so it needs to be available before skill_id has been set, i made it use /tmp and then move any files to the final location before `initialize` is called